jdk/make/CompileLaunchers.gmk

```diff
@@ -94,12 +94,16 @@ define SetupLauncher
   $1_LDFLAGS := $3
 
   ifeq ($(OPENJDK_TARGET_OS), linux)
-  # Set the image base address for zLinux 64 to 0x60000 for launchers,
+  # Set the image base address for p/zLinux 64 to 0x60000 for launchers,
   # allows compressedRefsShift to be 0 when -Xmx is set to 2040m or more.
+  # Also maximizes the address space available for the java heap.
   # / RTC PR 100052
     ifeq ($(OPENJDK_TARGET_CPU), s390x)
       $1_LDFLAGS += -Wl,-Ttext-segment=0x60000
     endif
+    ifeq ($(OPENJDK_TARGET_CPU), ppc64le)
+      $1_LDFLAGS += -Wl,-Ttext-segment=0x60000
+    endif
   endif
```

Comment:
The comments should be indented the same as the rest of the body of `ifeq`.
Since we want to treat p and z the same, the cases can be merged.
```
  ifeq ($(OPENJDK_TARGET_OS), linux)
    # Set the image base address for p/zLinux 64 to 0x60000 for launchers,
    # allows compressedRefsShift to be 0 when -Xmx is set to 2040m or more.
    # Also maximizes the address space available for the java heap.
    # /  RTC PR 100052
    ifneq (,$(filter ppc64le s390x, $(OPENJDK_TARGET_CPU)))
      $1_LDFLAGS += -Wl,-Ttext-segment=0x60000
    endif
  endif
```